File: app/src/main/java/com/travlog/android/apps/services/apirequests/SignInBody.java
package com.travlog.android.apps.services.apirequests;

public class SignInBody {

    public String userId;
    public String loginId;
    public String email;
    public String password;
    public String name;
    public String profilePicture;
    public String type;

    @Override
    public String toString() {
        return "XauthBody{" +
                "userId='" + userId + '\'' +
                ", loginId='" + loginId + '\'' +
                ", email='" + email + '\'' +
                ", password='" + password + '\'' +
                ", name='" + name + '\'' +
                ", profilePicture='" + profilePicture + '\'' +
                ", type='" + type + '\'' +
                '}';
    }
}

File: app/src/main/java/com/travlog/android/apps/services/apirequests/SignInBody.kt
package com.travlog.android.apps.services.apirequests

class SignInBody {

    var userId = ""
    var loginId = ""
    var email = ""
    var password = ""
    var name = ""
    var profilePicture = ""
    var type = ""

    override fun toString(): String {
        return "SignInBody(userId='$userId', loginId='$loginId', email='$email', password='$password', name='$name', profilePicture='$profilePicture', type='$type')"
    }
}
